DEPS

```diff
@@ -1,26 +1,30 @@
 use_relative_paths = True
 
 deps = {
-  "vendor/requests": "https://github.com/kennethreitz/requests@e4d59bedfd3c7f4f254f4f5d036587bcd8152458",
+  "components/playlist/browser/resources/playlist-component": {
```

Comment:
All existing deps are placed at either `vendor/` or `third_party/`. I'm not sure why but I'd like to get @bridiver opinion on whether putting one in an arbitrary place within components/ is ok.

Reply:
Yeah, that's what I was curious about too.

Reply:
why is this a separate repo in the first place?

Reply:
@bridiver  component updater uses it too to update the script. I suppose that repo could have brave-core as a dependency but if we change this to typescript then we'd need a brave-core script to "build" it outside the regular browser build vs a self contained build process in the independent repo. Doable.

Reply:
`vendor` is deprecated, but yes this should go in `brave/third_party/playlist-component`